### ISSUE TITLE
Failing to build ortools from source with SCIP 7.0.0

### ISSUE THREAD

**Issue by brianmunson** [google/or-tools#1953]
Here is a Dockerfile

```Dockerfile
FROM debian:10.3-slim

COPY scipoptsuite-7.0.0.tgz .

RUN apt-get -yy update \
    && apt-get -yy upgrade \
    && apt-get -yy install \
    git \
    wget \
    swig \
    cmake \
    libtool \
    autoconf \
    libbliss2 \
    zlib1g-dev \
    pkg-config \
    python3-pip \
    lsb-release \
    libbliss-dev \
    libncurses5-dev \
    libreadline-dev \
    build-essential \
    libboost-all-dev \
    && tar -xzvf scipoptsuite-7.0.0.tgz

RUN make \
    TPI=tny \
    GMP=false \
    ZIMPL=false \
    READLINE=false \
    USRCFLAGS=-fPIC \
    USRCXXFLAGS=-fPIC \
    USRCPPFLAGS=-fPIC \
    -j8 -C ./scipoptsuite-7.0.0 \
    && make install INSTALLDIR=/scip -j8 -C ./scipoptsuite-7.0.0

ENV UNIX_SCIP_DIR="/scip"

RUN git clone --depth 1 https://github.com/google/or-tools

RUN make -j8 -C ./or-tools third_party \
    && make -j8 -C ./or-tools python \
    && make -j8 -C ./or-tools install_python
```
After `make python` if I run `make test_python` I get the following near the end of the output:

```
/usr/bin/ld: /scip/lib/libscip.a(lp.o): relocation R_X86_64_PC32 against symbol `SCIProwComp' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /scip/lib/libscip.a(nlp.o): relocation R_X86_64_PC32 against symbol `stdout@@GLIBC_2.2.5' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /scip/lib/libscip.a(var.o): relocation R_X86_64_PC32 against symbol `SCIPvarComp' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /scip/lib/libscip.a(xmlparse.o): relocation R_X86_64_PC32 against symbol `stderr@@GLIBC_2.2.5' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /scip/lib/libscip.a(cons_countsols.o): relocation R_X86_64_PC32 against symbol `SCIPdialogExecCount' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /scip/lib/libscip.a(cons_superindicator.o): relocation R_X86_64_PC32 against symbol `SCIPdialogExecChangeMinUC' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /scip/lib/libscip.a(dialog_default.o): relocation R_X86_64_PC32 against symbol `SCIPdialogExecMenu' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /scip/lib/libscip.a(reader_opb.o): relocation R_X86_64_PC32 against symbol `stderr@@GLIBC_2.2.5' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /scip/lib/libscip.a(bandit_epsgreedy.o): relocation R_X86_64_PC32 against symbol `SCIPbanditResetEpsgreedy' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /scip/lib/libscip.a(bandit_exp3.o): relocation R_X86_64_PC32 against symbol `SCIPbanditResetExp3' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /scip/lib/libscip.a(bandit_ucb.o): relocation R_X86_64_PC32 against symbol `SCIPbanditResetUcb' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /scip/lib/libscip.a(dialog.o): relocation R_X86_64_PC32 against undefined symbol `rl_library_version' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: /scip/lib/libscip.a(disp.o): relocation R_X86_64_PC32 against symbol `SCIPparamChgdDispActive' can not be used when making a shared object; recompile with -fPIC
/usr/bin/ld: final link failed: nonrepresentable section on output
collect2: error: ld returned 1 exit status
make: *** [makefiles/Makefile.cpp.mk:177: lib/libortools.so] Error 1
```

I am not beholden to the base image for this build, but Dockerizing this is required.

The SCIP installation appears to be working, as evidenced by a successful `make test`.

**Comment by Mizux** [comment]
Thx for the feedback, last time I've tested it was SCIP 6.0.1
https://github.com/google/or-tools/issues/1848#issuecomment-576150025

AFAIK SCIP Makefile or CMake are usually broken rearding -fPIC management. I'll try to take a look at it...

BTW thanks for the Dockerfile...

**Comment by brianmunson** [comment]
one potential problem with my Dockerfile is that I neglected to include the same flags in `make install` as i did in the `make` step for SCIP. including them all still results in an error in `make test_python` however:

```
ImportError: /or-tools/ortools/algorithms/../gen/ortools/algorithms/../../../../lib/libortools.so: undefined symbol: _threadnumber
```

**Comment by Mizux** [comment]
Should be fixed on master which contains SCIP 7.0.1
